aiida_workgraph/utils/analysis.py

```diff
@@ -294,6 +291,9 @@ def check_diff(
         from node_graph.analysis import DifferenceAnalysis
 
         wg1 = self.get_wgdata_from_db(restart_process)
+        # also make a alias for nodes
+        wg1["nodes"] = wg1["tasks"]
+        self.wgdata["nodes"] = self.wgdata["tasks"]
```

Comment:
why do you not pop here as below?

Reply:
Thanks! Keeping both `tasks` and `nodes` will not raise an error in `DifferenceAnalysis`. But for consistency, I changed it to be the same as the code below.